[2024-07-24T09:53:31.330+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T09:53:31.523+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Fraud.gRPC_connection scheduled__2024-07-24T04:25:00+00:00 [queued]>
[2024-07-24T09:53:31.606+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Fraud.gRPC_connection scheduled__2024-07-24T04:25:00+00:00 [queued]>
[2024-07-24T09:53:31.607+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-24T09:53:31.705+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): gRPC_connection> on 2024-07-24 04:25:00+00:00
[2024-07-24T09:53:31.749+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=522) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T09:53:31.758+0000] {standard_task_runner.py:64} INFO - Started process 575 to run task
[2024-07-24T09:53:31.778+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_Fraud', 'gRPC_connection', 'scheduled__2024-07-24T04:25:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/server_DAG.py', '--cfg-path', '/tmp/tmp8zfn2p_b']
[2024-07-24T09:53:31.804+0000] {standard_task_runner.py:91} INFO - Job 122: Subtask gRPC_connection
[2024-07-24T09:53:31.965+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_Fraud.gRPC_connection scheduled__2024-07-24T04:25:00+00:00 [running]> on host 0542575fe516
[2024-07-24T09:53:32.226+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='fraud_detector' AIRFLOW_CTX_DAG_ID='ETL_Fraud' AIRFLOW_CTX_TASK_ID='gRPC_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T04:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-24T04:25:00+00:00'
[2024-07-24T09:53:32.231+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T09:53:32.238+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T09:53:32.243+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "gRPC_connection"']
[2024-07-24T09:53:32.271+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T09:53:32.285+0000] {subprocess.py:93} INFO - gRPC_connection
[2024-07-24T09:53:32.285+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T09:53:32.296+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T09:53:32.399+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_Fraud, task_id=gRPC_connection, run_id=scheduled__2024-07-24T04:25:00+00:00, execution_date=20240724T042500, start_date=20240724T095331, end_date=20240724T095332
[2024-07-24T09:53:32.499+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T09:53:32.544+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-24T09:53:32.547+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
